config.js

```diff
@@ -1,11 +1,17 @@
+/*
+@hazeltonbw When running `npm run create-db` there are no enviornment variables.
+By using the npm package `dotenv` we can use the .env file at root. You may not want to do it this way,
+but either way the npm script doesn't seem to be picking up the .env in root.
+*/
+require("dotenv").config();
```

Comment:
This seems better than having to specify how to use the dotenv file in your package.json scripts